daemons/fenced/fenced_remote.c

```diff
@@ -1178,7 +1178,14 @@ create_remote_stonith_op(const char *client, xmlNode *request, gboolean peer)
     op->state = st_query;
     op->replies_expected = fencing_active_peers();
     op->action = crm_element_value_copy(dev, F_STONITH_ACTION);
-    op->originator = crm_element_value_copy(dev, F_STONITH_ORIGIN);
+
+    /* The node initiating the stonith operation. If an operation is relayed,
```

Comment:
Kind of an awkward and arbitrary place to document this versus in `xml_names_internal.h`, since there are multiple functions that add the origin and multiple functions that check the origin. I understand the reasoning though.

I left `PCMK__XA_ROUTER_NODE`'s documentation in `xml_names_internal.h` for now, for the same reason -- it's used in so many places, there's no clear central place to put it. And it was added in a previous PR.